docs/book/src/user/quick-start.md

```diff
@@ -339,6 +454,24 @@ kubectl --kubeconfig=./capi-quickstart.kubeconfig \
   apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
 ```
 
+<aside class="note warning">
+<h1>Action Required (Docker provider v0.2.0 only)</h1>
+
+A [known issue](https://github.com/kubernetes-sigs/kind/issues/891) affects Calico with the Docker provider v0.2.0. After you deploy Calico, apply this patch to work around the issue:
```

Comment:
perhaps we can use a different CNI? maybe weave?

Reply:
Weave would allow for removing the need to override Cluster.Spec.ClusterNetwork as well.

Reply:
Filed https://github.com/kubernetes-sigs/cluster-api/issues/1533